webui/react/src/pages/ExperimentDetails/ExperimentSingleTrialTabs.tsx

```diff
@@ -269,17 +269,19 @@ const ExperimentSingleTrialTabs: React.FC<Props> = ({
         key: TabType.Checkpoints,
         label: 'Checkpoints',
       });
-      items.push({
-        children: (
-          <ExperimentCodeViewer
-            experiment={experiment}
-            selectedFilePath={settings.filePath}
-            onSelectFile={handleSelectFile}
-          />
-        ),
-        key: TabType.Code,
-        label: 'Code',
-      });
+      if (experiment.modelDefinitionSize !== 0) {
```

Comment:
in the ERD meeting, we had 2 options; hide tab or disable tab, and disable tab is the recommended one. Did we choose to hide tab?
We can actually disable tab already if you pass `disabled: true` props btw

Reply:
fixed